HypVINN/utils/img_processing_utils.py

```diff
@@ -146,14 +242,21 @@ def get_clean_mask(segmentation, optic=False):
     return clean_seg, labels_cc, savemask
 
 
-def get_clean_labels(segmentation):
+def get_clean_labels(segmentation: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
     """
     Function to find the largest connected component of the segmentation.
```

Comment:
Get clean labels by removing non-connected components from a dilated mask and any connected component with a size less than 3